frontend/public/service-worker.js

```diff
@@ -40,7 +40,6 @@ self.addEventListener('activate', event => {
 
 // Network falling back to the cache
 self.addEventListener('fetch', function(event) {
-  console.log('Fetch', event.request)
```

Comment:
is this one not also being replaced by a Logger call?

Reply:
nah, that was a leftover log from when I was getting the PWA code working, slipped through. I meant to comment on that in the PR but forgot!

Reply:
👍